.github/workflows/build.yml

```
@@ -0,0 +1,102 @@
+name: moonlight-android
+
+on: [push, pull_request]
```

```suggestion
on:
  pull_request:
    branches:
      - master
    types:
      - opened
      - synchronize
      - reopened
  push:
    branches:
      - master
  workflow_dispatch:

concurrency:
  group: "${{ github.workflow }}-${{ github.ref }}"
  cancel-in-progress: true
```

A little more explicit about the events and branches that this will run on, and provide a `workflow_dispatch` event (manual run).

Also, adds the workflow concurrency I mentioned previously.